tabs/tab_logit.py

```
@@ -1140,6 +1143,13 @@ def out_sg_table():
             return render.DataGrid(df_res[available_cols].round(4))
         return None
 
+    @render.ui
+    def out_sg_missing_report():
```

Ruff (0.14.13)

Missing return type annotation for private function `out_sg_missing_report`

(ANN202)